### PR TITLE
Autoconnect

### PR DESCRIPTION
**Pull request by bond95**
@mareklibra Please review

### CODE REVIEW

File: src/components/Checkbox.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React, { PropTypes } from 'react'
+
+const Checkbox = ({ checked, label, onClick }) => {
+  const onChange = (event) => {
+    onClick(event.target.checked)
+  }
+  console.log('checked', checked)
+
+  // let checked = config.get('filter') ? 'checked' : ''
```
Comment (mareklibra) [inline src/components/Checkbox.js]: leftover?

File: src/components/Options/ConsoleOptions.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  }
+}
+
+ConsoleOptions.propTypes = {
+  options: PropTypes.object.isRequired,
+  onSave: PropTypes.func.isRequired,
+  open: PropTypes.bool,
+}
+
+export default ConsoleOptions
```
Comment (mareklibra) [inline src/components/Options/ConsoleOptions.js]: new line, please

File: src/index.js [hunk 3]
```diff
@@ -2,6 +2,8 @@ export { default as VmDisks } from './components/VmDisks'
 
 export { default as VmIcon } from './components/VmIcon'
 export { default as VmStatusIcon } from './components/VmStatusIcon'
+export { default as Checkbox } from './components/Checkbox'
 
+export * from './components/Options'
 export * from './vm-status'
-export * from './helpers'
+export * from './helpers'
```
Comment (mareklibra) [inline src/index.js]: new line

**Review by mareklibra — CHANGES_REQUESTED** [review]
Looks good, just minor clean-ups

@bond95 , please rebase on top of clean master branch

**Comment by bond95** [comment]
@mareklibra fixed, please review

**Review by mareklibra — APPROVED** [review]

**Pull request merged**